incubator/fluentd-cloudwatch/README.md

```diff
@@ -53,6 +56,7 @@ The following tables lists the configurable parameters of the Fluentd Cloudwatch
 | `hostNetwork`                   | Host network                               | `false`                                                    |
 | `annotations` (removed for now) | Annotations                                | `nil`                                                      |
 | `awsRegion`                     | AWS Cloudwatch region                      | `us-east-1`                                                |
+| `awsRole`                       | AWS IAM Role To Use                        | `nil`                                                      |
```

Comment:
please add documentation for `rbac.create` and `rbac.serviceAccountName`

Reply:
@icereval thanks for the pointers, this the rbac.create and rbac.serviceAccountName were added.